### PR TITLE
Use the project name when creating a new project file locally

### PR DESCRIPTION
**Pull request by D8H**

### CODE REVIEW

File: newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js [hunk 1]
```diff
@@ -247,9 +250,12 @@ export const onRenderNewProjectSaveAsLocationChooser = ({
     : newProjectsDefaultFolder
     ? newProjectsDefaultFolder
     : '';
+  const projectFileName = projectName
+    ? cleanUpIdentifier(projectName, 1, 60) + '.json'
+    : 'game.json';
```
Comment (D8H) [inline newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js:255]: Should non-latin characters be allowed?
It may work and be more user-friendly but it can probably cause issues too.

Comment (ClementPasteau) [reply newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js:255]: I think we can only prevent "non-allowed" characters, that can cause issues on some OS (windows in particular)
We've done something similar when generating the game:

https://github.com/4ian/GDevelop/blob/81bce617836326fd8d0346de8efdb499f4dbd042/GDJS/GDJS/IDE/ExporterHelper.cpp#L72-L93

**Review by ClementPasteau — APPROVED** [review]
nice!
Maybe a small unit test on that function to ensure we haven't missed the edge cases? (only forbidden characters, 2 consecutive, syrilic, etc...)

**Comment by ClementPasteau** [comment]
> Why not but the function is very straightforward it would almost be like unit testing `String.replace`.

up to you!
